spec/std/isa/ext/Zvqdotq.yaml

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) Kallal Mukherjee and/or its contributors.
```

Comment:
This is an odd phrasing. If it was just you that created this content, then just put yourself as the copyright owner.

Same comment for all of the files in this PR.